awx_collection/tests/integration/targets/tower_project/tasks/main.yml

```diff
@@ -9,245 +9,245 @@
-  tower_job_template:
-    name: "{{ jt1 }}"
-    project: "{{ project_name3 }}"
-    inventory: "Demo Inventory"
-    scm_branch: master
-    playbook: debug.yml
-
-- name: Launch "{{ jt1 }}"
-  tower_job_launch:
-    job_template: "{{ jt1 }}"
-  register: result
-
-- assert:
-    that:
-      - result is changed
-
-- name: "wait for job {{ result.id }}"
-  tower_job_wait:
-    job_id: "{{ result.id }}"
-  register: job
-
-- assert:
-    that:
-      - job is successful
-
-- name: Delete the test job_template
-  tower_job_template:
-    name: "{{ jt1 }}"
-    project: "{{ project_name3 }}"
-    inventory: "Demo Inventory"
-    state: absent
-
-- name: Delete the test project 3
-  tower_project:
-    name: "{{ project_name3 }}"
-    organization: Default
-    state: absent
-
-- name: Delete the test project 2
-  tower_project:
-    name: "{{ project_name2 }}"
-    organization: "{{ org_name }}"
-    state: absent
-
-- name: Delete the SCM Credential
-  tower_credential:
-    name: "{{ scm_cred_name }}"
-    organization: Default
-    kind: scm
-    state: absent
-  register: result
-
-- assert:
-    that:
-      - result is changed
-
-- name: Delete the test project 1
-  tower_project:
-    name: "{{ project_name1 }}"
-    organization: Default
-    state: absent
-  register: result
-
-- assert:
-    that:
-      - result is changed
-
-- name: Delete credential
-  tower_credential:
-    kind: scm
-    name: "{{ cred_name }}"
-    organization: "{{ org_name }}"
-    state: absent
-  register: result
-
-- assert:
-    that:
-      - result is changed
-
-- name: Delete the organization
-  tower_organization:
-    name: "{{ org_name }}"
-    state: absent
-  register: result
-
-- assert:
-    that:
-      - result is changed
+- block:
+    - name: Create an SCM Credential
+      tower_credential:
+        name: "{{ scm_cred_name }}"
+        organization: Default
+        kind: scm
+      register: result
+
+    - assert:
+        that:
+          - result is changed
+
+    - name: Create a git project without credentials and wait
+      tower_project:
+        name: "{{ project_name1 }}"
+        organization: Default
+        scm_type: git
+        scm_url: https://github.com/ansible/test-playbooks
+        wait: true
+      register: result
+
+    - assert:
+        that:
+          - result is changed
+
+    - name: Recreate the project to validate not changed, currenty reports inaccurate warning
+      tower_project:
+        name: "{{ project_name1 }}"
+        organization: Default
+        scm_type: git
+        scm_url: https://github.com/ansible/test-playbooks
+        wait: false
+      register: result
+      ignore_errors: true
+
+    - debug:
+        var: result
+    - assert:
+        that:
+          - result is changed
+          - "'The field scm_track_submodules of project' in result.warnings[0]"
+          - "'inaccurately report task is changed.' in result.warnings[0]"
```

Comment:
Result is `not changed` and the below assertions no longer apply :heavy_check_mark: